roles/artifacts/tasks/edpm.yml

```diff
@@ -23,31 +23,32 @@
   when:
     - not cifmw_is_nested_virt | default(false) | bool
   block:
-    - name: Check for CI env directory
-      register: network_env_dir
-      ansible.builtin.stat:
-        path: /etc/ci/env
+    - name: Slurp zuul inventory file
+      ansible.builtin.slurp:
+        path: "{{ ansible_user_dir }}/ci-framework-data/artifacts/zuul_inventory.yml"
+      register: _inv
 
-    - name: Extract data from network env file if available
-      when:
-        - network_env_dir.stat.exists
-      block:
-        - name: Load network env file
-          ansible.builtin.include_vars:
-            dir: /etc/ci/env/
+    - name: Filter compute ips from reproducer/zuul generated inventory
+      vars:
+        _compute_data: "{{ _inv['content'] | b64decode | from_yaml }}"
+      ansible.builtin.set_fact:
```

Comment:
No need to create an entire new task to just set a fact that can be a variable of this task https://github.com/openstack-k8s-operators/ci-framework/pull/1360/files#diff-8bc3f0e951dc1302fb2cc860f49de1369e43a6c4fc7b55fca0da1df484cbf327R41

Reply:
Done